### PR TITLE
Refactor cookie banner

### PR DESCRIPTION
**Pull request by LidorLapid**

### CODE REVIEW

File: assets/js/modules/legacy.js [hunk 1]
```diff
@@ -1681,14 +1681,27 @@ var kcc = {
...
+              <p class="cookie-notification__text">We use <a class="cookie-notification__link" href="/about-the-council/about-the-website/cookies">cookies</a><span class="cookie-notification__extended-text"> to improve your experience.</span>
+                <a class="cookie-notification__close-button">
+                  <span class="cookie-notification__close-icon" aria-hidden="true">&times;</span>
+
+                  <span class="show-for-sr">Close cookie notification</span>
+                </a>
+              </p>
+            </div>
+          </div>
+        </div>`;
```
Comment (wellhairy) [inline assets/js/modules/legacy.js]: Back ticks are literally the best!

File: assets/js/modules/legacy.js [hunk 2]
```diff
@@ -1699,9 +1712,9 @@ var kcc = {
...
     },
 
-    closeBanner: function(event) {
-      event.preventDefault();
-      $('#cookie-law').fadeOut(function() {
+    closeBanner: function(e) {
+      e.preventDefault();
+      $('.cookie-notification').fadeOut(function() {
         $(this).remove();
       });
```
Comment (wellhairy) [inline assets/js/modules/legacy.js]: Do you know if we're detaching the event listener, It doesn't look like it?

File: assets/scss/partials/_mixins.scss [hunk 3]
```diff
@@ -9,6 +9,15 @@
...
 }
 
+@mixin show-for-sr() {
+    position: absolute;
+    left: -10000px;
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+}
```
Comment (wellhairy) [inline assets/scss/partials/_mixins.scss]: Should this just be a layout class?

File: assets/js/modules/legacy.js (hunk 2 above)
Comment (LidorLapid) [inline assets/js/modules/legacy.js]: Nope, we aren't currently. This was purely a BEM refactor at this stage -- I suggest we revisit the JS when we refactor into ES6.

File: assets/scss/partials/_mixins.scss (hunk 3 above)
Comment (LidorLapid) [inline assets/scss/partials/_mixins.scss]: .show-for-sr is a default Foundation 6 class, but it's not easily usable inside a media query as we don't want to be using @extend, hence I thought a mixin would come in handy.

**Comment by natmunckton** [comment]
LGTM

**Pull request merged**